### PR TITLE
Create fishy.json

### PR DESCRIPTION
**Pull request by skibidiloser**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/fishy.json
Comment (CuteDog5695) [inline domains/fishy.json:1]: Json file literally doesn't have anything

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/fishy.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "<skibidiloser>",
+    "email": "<auroralindgren23@gmail.com>"
```
Comment (CuteDog5695) [inline domains/fishy.json:4]:
```suggestion
    "username": "skibidiloser",
    "email": "auroralindgren23@gmail.com"
```

File: domains/fishy.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  }
+}
+{
+  "record": {
+    "CNAME": "<skibidiloser>.github.io"
```
Comment (CuteDog5695) [inline domains/fishy.json:9]:
```suggestion
    "CNAME": "skibidiloser.github.io"
```

Comment (CuteDog5695) [reply domains/fishy.json:9]: Also site gives 404 error, if you have the site hosted then please send a preview of the website

**Review by iostpa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**